src/main/java/ua/kpi/iasa/IASA_Organiser/data/impl/EventStore.java

```diff
@@ -2,7 +2,11 @@
 
 import ua.kpi.iasa.IASA_Organiser.model.Event;
 
-public class EventStore {  //smth like database/it`s foundation
+/**
+ * @deprecated It was useful at first lab, but now useless
```

Comment:
Change java docs comment to
"Class was used by {@link DefaultArrayDataManager}, but not that DataManager is deprecated and you don't need to use {@link EventStore} class"

Reply:
Fixed